build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.1.0'
+version '1.1.1'
```

Comment:
this isn't a patch version bump, you've added more features, and if the constructor isn't restored in GeneratePinRequest this is a breaking change

I would suggest doing at least a minor version bump, we shouldn't require a major version for this change